.gitignore

```diff
@@ -0,0 +1,177 @@
+# Created by https://www.toptal.com/developers/gitignore/api/macos,node
+# Edit at https://www.toptal.com/developers/gitignore?templates=macos,node
```

Comment:
I think you should include the `VisualStudioCode` template so that it ignores any IDE related directories/files.